src/modules/logger/logger.cpp

```diff
@@ -525,13 +525,13 @@ void Logger::add_default_topics()
 	add_topic("manual_control_setpoint", 200);
 	add_topic("mission");
 	add_topic("mission_result");
-	add_topic("optical_flow", 50);
-	add_topic("position_controller_status", 500);
+	add_topic("position_controller_status", 1000);
 	add_topic("position_setpoint_triplet", 200);
 	add_topic("radio_status");
 	add_topic("rate_ctrl_status", 200);
 	add_topic("sensor_combined", 100);
 	add_topic("sensor_preflight", 200);
+	add_topic("sensor_selection", 1000);
```

Comment:
Does this get published only when it changes? I think we should strive for zero delay logging for this critical item and rather ensure it doesn't publish at a crazy rate to gate it.

Reply:
Yes, it should only be publishing on change.

Reply:
Removed the sensor_selection interval.